### PR TITLE
Farm Config: V3

### PR DESCRIPTION
**Pull request by hachiojidev**

### CODE REVIEW

File: src/config/constants/farms.ts [hunk 1]
```diff
@@ -12,6 +12,2441 @@ const farms: FarmConfig[] = [
...
+    lpSymbol: 'IOTX-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xc13aA76AAc067c86aE38028019F414D731b3D86A',
+    },
+    token: tokens.iotx,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 197,
```
Comment (ChefHutch) [inline src/config/constants/farms.ts:570]: PID should be 308 here Haci 👌

File: src/config/constants/farms.ts [hunk 2]
```diff
@@ -12,6 +12,2441 @@ const farms: FarmConfig[] = [
...
+    },
+    token: tokens.bry,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 302,
+    lpSymbol: 'ZEE-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x21dD71aB78EDE3033c976948f769D506E4F489eE',
```
Comment (ChefHutch) [inline src/config/constants/farms.ts:624]: I think this is the BRY-BNB address, it should be `0x8e799cB0737525CeB8A6C6Ad07f748535fF6377B`

File: src/config/constants/farms.ts [hunk 3]
```diff
@@ -12,6 +12,2441 @@ const farms: FarmConfig[] = [
...
+    },
+    token: tokens.dodo,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 303,
+    lpSymbol: 'BRY-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x4Fd6D315bEf387fAD2322fbc64368fC443F0886D',
```
Comment (ChefHutch) [inline src/config/constants/farms.ts:614]: I think this is the SWINGBY-BNB address, it should be `0x21dD71aB78EDE3033c976948f769D506E4F489eE`

File: src/config/constants/farms.ts [hunk 4]
```diff
@@ -12,6 +12,2441 @@ const farms: FarmConfig[] = [
...
+    lpAddresses: {
+      97: '',
+      56: '0xCE383277847f8217392eeA98C5a8B4a7D27811b0',
+    },
+    token: tokens.yfi,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 267,
+    lpSymbol: 'YFII-BNB LP',
```
Comment (ChefHutch) [inline src/config/constants/farms.ts:981]: Is this a duplicate? Or deliberately have two different LPs with the same pid/address?

**Review by ChefHutch — APPROVED** [review]
👍

File: src/config/constants/farms.ts [hunk 5]
```diff
@@ -12,6 +12,2431 @@ const farms: FarmConfig[] = [
...
+    lpSymbol: 'BUSD-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x58F876857a02D6762E0101bb5C46A8c1ED44Dc16',
+    },
+    token: tokens.busd,
+    quoteToken: tokens.wbnb,
+  },
+  /**
+   * All farms below here are from v1 and are to be set to 0x
```
Comment (Chef-Cheems) [inline src/config/constants/farms.ts:1130]: @hachiojidev  v1 farms got duplicated for some reason, we have them already down below.

Comment (hachiojidev) [reply src/config/constants/farms.ts]: @Chef-Cheems Nice catch

**Review by Chef-Chungus — APPROVED** [review]

**Pull request merged**